transfer/transfer.go

```diff
@@ -24,12 +17,6 @@ const (
 // name and dir are to provide context if one func implements many instances
 type NewTransferAdapterFunc func(name string, dir Direction) TransferAdapter
```

Comment:
Just need to update the comments in this area to point people at the new `Manifest` type to find registration functions etc